templates/devshells/rust/nixel/contracts.ncl

```diff
@@ -186,7 +186,7 @@ from the Nix world) or a derivation defined in Nickel.
       system
         | doc "The system to build the package on."
         | System
-        | default = {arch = "x86_64", os = "linux"},
+        | optional,
```

Comment:
Worth expanding a bit the doc to explain what it means for this system to be absent